lesson 2/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@700&family=Mulish&display=swap" rel="stylesheet">
+    <title>Logoipsum</title>
+</head>
+<body>
+    <header class = "header">
+        <div class = "container">
+            <div class = "header__inner">
+                <div class = "header__logo"><img src="Logo Color.png" alt = "logo"></div>
+                <nav class = "header__nav">
+                    <a class = "nav__link" href = "#">Home</a>

```

Comment:
<nav class = "header__nav">
your may write like <nav class = "nav"> it's correct or if you need two class 
<nav class = "header__nav nav">
but it's not mistake